arelle/ModelDocument.py

```diff
@@ -1328,11 +1328,13 @@ def inlineXbrlDiscover(self, htmlElement):
                 if inlineElement.get("target") == ixdsTarget:
                     self.schemaLinkbaseRefsDiscover(inlineElement)
                     xmlValidate(self.modelXbrl, inlineElement) # validate instance elements
+        # validate resources only if no possibility of multi-document ixds for which ix:references not encountered yet
+        if all(False for pluginMethod in pluginClassMethods("ModelDocument.SelectIxdsTarget")):
```

Comment:
This needs to call the plugin functions?

```suggestion
        if all(pluginMethod(self.modelXbrl) is False for pluginMethod in pluginClassMethods("ModelDocument.SelectIxdsTarget")):
```

Reply:
No, in this case the if statement should only run if there were no ModelDocument.SelectIxdsTarget methods loaded and not execute that plugin method at this point.  If ModelDocument.SelectIxdsTarget exists, then that plugin method runs later on, after all ixds modelDocuments have completely been loaded and their ix:resources (and possibly selected target ix:references) are ready to be processed